src/main/java/racingcar/IOClass.java

```diff
@@ -0,0 +1,71 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class IOClass {
+
+    public static Car[] getName() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+        str = str.replace(" ", "");
+        String[] carNames = str.split(",");
+        Car[] cars = new Car[carNames.length];
+        for (int i = 0; i < cars.length; i++) {
+            cars[i] = new Car(carNames[i]);
+        }
+
+        return cars;
+    }
+
+    public static boolean checkCarName(Car[] cars) {
+        for (int i = 0; i < cars.length; i++) {
+            if (cars[i].getName().length() > 5) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static int getTimes() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        Scanner sc = new Scanner(System.in);
+        int times = sc.nextInt();
+        return times;
+    }
+
+    public static void printCurrentPosition(Car[] cars) {
+        for (int i = 0; i < cars.length; i++) {
+            System.out.print(cars[i].getName() + " : ");
+            for (int j = 0; j < cars[i].getPosition(); j++) {
+                System.out.print("-");
+            }
+            System.out.println();
+        }
+    }
+
+    public static void printWinners(Car[] cars) {
+        int count = 0;
+        int maxScore = Race.getMaxScore(cars);
+        System.out.print("최종 우승자: ");
+        ArrayList<String> winnerNames = getWinnerNames(cars, maxScore);
+        for (int i = 0; i < winnerNames.size(); i++) {
+            if (count != 0) {
+                System.out.print(", ");
+            }
+            System.out.print(winnerNames.get(i));
+            count++;
+        }
+    }
+
+    static ArrayList<String> getWinnerNames(Car[] cars, int maxScore) {
+        ArrayList<String> winners = new ArrayList<>();
```

Comment:
```suggestion
        List<String> winners = new ArrayList<>();
```
Collection은 인터페이스(List, Map, Set)를 변수의 타입으로 지정하고 초기화를 할 때, 하위 구현체(ex. ArrayList, LinkedList, HashMap.... etc)로 초기화해야합니다! 이는 객체지향의 중요한 특징 중 하나인 다형성을 이용하기 위해서예요! 😃 

[다형성이란...](https://tecoble.techcourse.co.kr/post/2020-10-27-polymorphism/)
[다형성의 개념](http://www.tcpschool.com/java/java_polymorphism_concept)

또한 추가적으로 `getWinnerNames` 메서드는 IOClass 내부에서만 사용하기 때문에 `private` **접근 제한자**를 둬서 외부에서 호출하는 것을 막아주세요! 🤗